src/uct/cuda/gdr_copy/gdr_copy_md.c

```diff
@@ -267,9 +271,10 @@ uct_gdr_copy_rache_region_from_memh(uct_mem_h memh)
     return ucs_container_of(memh, uct_gdr_copy_rcache_region_t, memh);
 }
 
-static ucs_status_t
-uct_gdr_copy_mem_rcache_reg(uct_md_h uct_md, void *address, size_t length,
-                            unsigned flags, uct_mem_h *memh_p)
+UCS_PROFILE_FUNC(ucs_status_t, uct_gdr_copy_mem_rcache_reg,
```

Comment:
Seeing that the application is spending a lot of time in just registration (similar to the overhead that @nsakharnykh saw for join use case) may be useful but I guess you're suggesting this because rcache ops are already profiled? I do see `UCS_PROFILE_CALL` calls in rcache.c
If yes, I can remove it but if the application spent a lot of time in registration of both host and cuda buffers, then without profiling gdr_copy_mem_rcache_reg we wouldn't be able to distinguish the different ones.

Reply:
rcache ops are already profiled. also need to profile registration itself (when rcache has a lookup miss), not the lookup itself

Reply:
I've removed the profiling call